test/acceptance/tests/controller/controller_namespaces_test.go

```diff
@@ -134,6 +134,13 @@ func TestControllerNamespaces(t *testing.T) {
 					svcResolverEntry, ok := entry.(*api.ServiceResolverConfigEntry)
 					require.True(r, ok, "could not cast to ServiceResolverConfigEntry")
 					require.Equal(r, "bar", svcResolverEntry.Redirect.Service)
+
+					// proxy-defaults
+					entry, _, err = consulClient.ConfigEntries().Get(api.ProxyDefaults, "global", queryOpts)
```

Comment:
So your `Get()` call will need to be in the default consul namespace. this means removing `queryOpts` and using `nil` (which will default to default ns) or constructing a new one and explicitly setting it to `default`